### PR TITLE
Update Text Analytics C# SDK auto-generated code.

### PR DESCRIPTION
**Pull request by yangyuan**
Swagger PR: https://github.com/Azure/azure-rest-api-specs/pull/3491
Doc PR: https://github.com/MicrosoftDocs/azure-docs/pull/12442

Please hold for @assafi's review.

## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Review by assafi — APPROVED** [review]

File: src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.csproj [hunk 1]
```diff
@@ -6,7 +6,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.Language.TextAnalytics</PackageId>
     <Description>This client library provides access to the Microsoft Cognitive Services Language APIs.</Description>
-    <Version>2.0.0-preview</Version>
+    <Version>2.1.0-preview</Version>
     <AssemblyName>Microsoft.Azure.CognitiveServices.Language.TextAnalytics</AssemblyName>
     <PackageTags>Microsoft Cognitive Services;Cognitive Services;Cognitive Services SDK;Text Analytics API;Text Analytics;REST HTTP client;netcore451511</PackageTags>
     <PackageReleaseNotes>
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics/Microsoft.Azure.CognitiveServices.Language.TextAnalytics.csproj]: Please update the `PackageReleaseNotes` here

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks great apart from the minor comment

**Review by dsgouda — APPROVED** [review]
LGTM

**Comment by dsgouda** [comment]
Will merge on CIs passing

**Comment by yangyuan** [comment]
@dsgouda weird it failed because some package issue. I so rebased to newest psSdkJson6 to see if it works.

**Comment by yangyuan** [comment]
@dsgouda, it seems to be an issue in Batch:
c:\workspace\VSUpgradeForTest\src\SDKs\Batch\DataPlane\Azure.Batch.Unit.Tests\Azure.Batch.Unit.Tests.csproj : error NU1605: Detected package downgrade: Microsoft.NET.Test.Sdk from 15.0.0 to 15.0.0-preview-20170222-09. Reference the package directly from the project to select a different version.

**Comment by dsgouda** [comment]
@yangyuan kicked off another build. Will investigate if this fails

**Comment by yangyuan** [comment]
@dsgouda still falls, same error.
Seems that it was cause by the references to different Microsoft.NET.Test.Sdk versions.
Not related to Text Analytics because Text Analytics refer shared AzSdk.test.reference.props

**Comment by dsgouda** [comment]
@yangyuan We are fixing it now

**Pull request merged**